python/meterpreter/ext_server_stdapi.py

```diff
@@ -2524,7 +2525,7 @@ def stdapi_net_config_get_routes_via_osx_netstat():
             continue
         if destination == 'default':
             destination = all_nets
-        if re.match('link#\d+', gateway) or re.match('([0-9a-f]{1,2}:){5}[0-9a-f]{1,2}', gateway):
+        if re.match('link#\\d+', gateway) or re.match('([0-9a-f]{1,2}:){5}[0-9a-f]{1,2}', gateway) or re.match('([0-9a-f]{1,2}.){5}[0-9a-f]{1,2}', gateway):
             gateway = all_nets[:-2]
```

Comment:
I think this existing behavior is odd to default to `0.0.0.0` or `::`

The context being, if `ifconfig` is output as:

```
netstat -rn
Routing tables

Internet:
Destination        Gateway            Flags        Netif Expire
...
192.168.85         link#17            UC         bridge1      !
...
```

It appears to the user in msfconsole as `0.0.0.0` instead of `link#17`:

```
meterpreter > route

IPv4 network routes
===================

    Subnet           Netmask          Gateway    Metric  Interface
    ------           -------          -------    ------  ---------
....
    192.168.85.0     255.255.255.255  0.0.0.0    0       bridge1
...
```

I haven't confirmed, but I'm guessing this weirdness is for Metasploit's pivoting support behind the scenes